### PR TITLE
feat: Badge Component

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Badge on icon
Badge on avatar

Please review.
Thanks

link https://nifty-ride-eb7903.netlify.app/

### CODE REVIEW

File: lib/base/variables.css [hunk 1]
```diff
@@ -48,6 +48,8 @@
   --fg3: #665c54;
```
Comment (ShreyashMahajan) [inline lib/base/variables.css:48]: here you can use a more understandable name.

**Review by ShreyashMahajan — APPROVED** [review]
code looks good to merge.

File: lib/components/avatar.css [hunk 2]
```diff
@@ -1,59 +1,47 @@
...
 .av,
 .av-sm,
-.av-md, 
-.av-lg, 
+.av-md,
+.av-lg,
 .av-xl {
-    border-radius: 100%;
+  border-radius: 100%;
 }
```
Comment (JantuDeb) [inline lib/components/avatar.css:12]: `.av {`
`  height: 4.3rem;`
`  width: 4.3rem;`
` border-radius:100%`
`}`
Then add this class to all avatars, so you don't need to select all the avatar classes to add border radius.

**Review by JantuDeb — APPROVED** [review]
Suggested some changes.

File: lib/components/avatar.css (hunk 2 above)
Comment (ozairkhan2810) [reply lib/components/avatar.css:12]: ok! got it
Thanks

File: lib/base/variables.css (hunk 1 above)
Comment (ozairkhan2810) [reply lib/base/variables.css:48]: sure! will do
Thanks

**Pull request merged**